stardis/radiation_field/opacities/opacities_solvers/base.py

```diff
@@ -449,13 +464,13 @@ def calc_alpha_line_at_nu(
         radiation=radiation,
     )
 
-    alpha_line_at_nu = np.zeros((no_depth_points, len(tracing_nus)))
+    alpha_line_at_nu = np.zeros((no_of_depth_points, len(tracing_nus)))
 
     for i in range(len(tracing_nus)):
         nu = tracing_nus[i].value
         delta_nus = nu - line_nus
 
-        for j in range(no_depth_points):
+        for j in range(no_of_depth_points):
```

Comment:
Very nitpicky, but would it be more descriptive to use a name like `depth_point_count`?